### PR TITLE
Adding support for Kudulite Bullseye

### PR DESCRIPTION
**Pull request by vijaysaayi**
In this PR,
- updating Oryx base image tag which has bullseye based oryx build image.
- Adding scripts needed to generate Bullseye Kudulite Image

This change will be cherry picked to releases/ant98 branch

### CODE REVIEW

File: localGenerateDockerFiles.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ done
 
 artifactStagingDirectory="output/DockerFiles"
 baseImageName="${oryxBaseImageName:="mcr.microsoft.com/oryx"}" 
-baseImageVersion="${oryxTagName:="20220329.1"}" # change me as needed
+baseImageVersion="${oryxTagName:="20220502.2"}" # change me as needed
```
Comment (surenderssm) [inline localGenerateDockerFiles.sh:33]: Is this intended?

File: GenerateDockerFiles/KuduLite/generateDockerfiles.sh [hunk 2]
```diff
@@ -82,6 +108,7 @@ function pullAppSvcRepo()
     git checkout $APP_SVC_REPO_BRANCH
     cd $ORIG_DIR
     chmod -R 777 $APP_SVC_REPO_DIR
+    echo
```
Comment (surenderssm) [inline GenerateDockerFiles/KuduLite/generateDockerfiles.sh:111]: NIT: Is this intended?

File: GenerateDockerFiles/KuduLite/generateDockerfiles.sh [hunk 3]
```diff
@@ -91,9 +118,21 @@ function pullAppSvcRepo()
     git checkout $DYN_INST_REPO_BRANCH
     cd $ORIG_DIR
     chmod -R 777 $DYN_INST_REPO_DIR
+    echo
```
Comment (surenderssm) [inline GenerateDockerFiles/KuduLite/generateDockerfiles.sh:121]: nit?

**Comment by surenderssm** [comment]
Looks good to me at a high level.

Please get it reviewed from the owner/expert of this repo as well.

File: localGenerateDockerFiles.sh (hunk 1 above)
Comment (vijaysaayi) [reply localGenerateDockerFiles.sh:33]: Yes. This tag of oryx has bullseye image

File: GenerateDockerFiles/KuduLite/generateDockerfiles.sh (hunk 3 above)
Comment (vijaysaayi) [reply GenerateDockerFiles/KuduLite/generateDockerfiles.sh:121]: echo ? This gives a line break so its easy to read the logs

File: GenerateDockerFiles/KuduLite/generateDockerfiles.sh (hunk 2 above)
Comment (vijaysaayi) [reply GenerateDockerFiles/KuduLite/generateDockerfiles.sh:111]: echo ? This gives a line break so its easy to read the logs

Comment (shpraka) [reply GenerateDockerFiles/KuduLite/generateDockerfiles.sh:111]: Shall we put a comment mentioning the purpose?

Comment (vijaysaayi) [reply GenerateDockerFiles/KuduLite/generateDockerfiles.sh:111]: Yes I will add this

File: CI.yml [hunk 4]
```diff
@@ -217,6 +217,25 @@ jobs:
       filesRootPath: GitRepo-DynInst
       imgTag: buster_$(Build.BuildNumber)
 
+- job: Job_BuildKuduLiteBullseyeImage
+  displayName: Build KuduLite Bullseye Image
+  dependsOn: Job_GenerateDockerFiles
+  pool:
+    vmImage: ubuntu-18.04
```
Comment (rabollin) [inline CI.yml:224]: May be in next iteration, can we switch to ubuntu 20x?

**Review by rabollin — APPROVED** [review]

**Comment by rabollin** [comment]
Approved

**Review by rabollin — APPROVED** [review]

**Comment by rabollin** [comment]
Approved

File: CI.yml (hunk 4 above)
Comment (vijaysaayi) [reply CI.yml:224]: Sure Radhika. We will take this up along with ImageBuilder v2 changes

**Pull request merged**